docs/sources/microsoft-365/msft-teams/msft-teams.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 endpoints:
-  - pathTemplate: "/{apiVersion}/teams"
+  - pathTemplate: "/v1.0/teams"
```

Comment:
Following #735, forcing to use v1.0 in Teams instead of being part of path template